ScnSoft-Education/LeetCodeTraining/RemoveNthNodeFromEndOfList18/RemoveNthNodeFromEndOfList.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace LeetCodeTraining.RemoveNthNodeFromEndOfList18
+{
+    public class ListNode
+    {
+        public int val;
+        public ListNode next;
+        public ListNode(int val = 0, ListNode next = null)
+        {
+            this.val = val;
+            this.next = next;
+        }
+    }
+    public class Solution
```

Comment:
add an empty line before declaring the class

Reply:
adding an empty line